tests/test-network-transport/src/test/kotlin/testnetworktransport/QueueTestNetworkTransportHandlerTest.kt

```diff
@@ -7,29 +7,34 @@ import com.apollographql.apollo3.api.Error
 import com.apollographql.apollo3.testing.QueueTestNetworkTransport
 import com.apollographql.apollo3.testing.enqueueTestNetworkError
 import com.apollographql.apollo3.testing.enqueueTestResponse
-import com.apollographql.apollo3.testing.runTest
+import com.apollographql.apollo3.testing.internal.runTest
 import com.benasher44.uuid.uuid4
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestScope
 import testnetworktransport.test.GetHeroQuery_TestBuilder.Data
 import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
 import kotlin.test.assertTrue
 
+@OptIn(ExperimentalCoroutinesApi::class)
 class QueueTestNetworkTransportHandlerTest {
   private lateinit var apolloClient: ApolloClient
 
-  private fun setUp() {
+  private fun TestScope.setUp() {
     apolloClient = ApolloClient.Builder()
         .networkTransport(QueueTestNetworkTransport())
+        .dispatcher(StandardTestDispatcher(testScheduler))
```

Comment:
Why do we need to change the dispatcher here?

Reply:
So it was not needed after all?

Reply:
Sorry for not replying earlier! In the previous version which was using `kotlinx.coroutines.test.runTest` this was to make the client use the `testScheduler` - without it, turbine's awaitXxx methods time out but I must admit I'm not sure why!

Now we've reverted to using `runBlocking` under the hood, so need for this.